notebooks/README.md

```diff
@@ -0,0 +1,35 @@
+### Notebooks
+
+This folder contains various reproducible and interactive Jupyter notebooks created for the project. These notebooks are also available for viewing on our public [JupyterHub instance](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login).
+The notebook organizational structure is described in detail below.
+
+#### Data Sources
+
+The notebooks defined in the [`data-sources`](notebooks/data-sources) folder, explore and analyze the [Sippy](https://github.com/openshift/sippy) and [TestGrid](https://github.com/GoogleCloudPlatform/testgrid) data sources.
```

Comment:
The link to `data-sources` (and also links to other directories and notebooks) seem to give a 404. Is that just me or is there an extra `notebooks` in the link path?
```suggestion
The notebooks defined in the [`data-sources`](data-sources) folder, explore and analyze the [Sippy](https://github.com/openshift/sippy) and [TestGrid](https://github.com/GoogleCloudPlatform/testgrid) data sources.
```

Reply:
ah yes, links updated :+1: